dist/stat_data.py

```diff
@@ -1086,6 +1086,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_read_checkpoint', 'pages read into cache by checkpoint'),
     CacheStat('cache_read_deleted', 'pages read into cache after truncate'),
     CacheStat('cache_read_deleted_prepared', 'pages read into cache after truncate in prepare state'),
+    CacheStat('cache_read_delta', 'pages read into cache as deltas'),
```

Comment:
What is the intended meaning of this stat? I think from the description it's unclear, since there might be multiple deltas per page.

If the current code is correct, I think the description should be more like `number of pages read that had deltas attached`.

If it's the number of deltas read (and this seems more useful to me?), then we probably ought to call it something like `number of deltas read into cache`. (And then change the code to do a `WT_STAT_CONN_DSRC_INCRV`.)